package.json

```diff
@@ -100,15 +100,15 @@
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
     "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#398bf7c6a6d37f229a41d92bd7a4324c0fd32849",
-    "expo": "^49.0.0",
+    "expo": "^50.0.0-preview.7",
     "expo-image": "1.8.1",
```

Comment:
Is there a reason why this is not bumped to 1.10.1?

Reply:
reverted in https://github.com/Expensify/App/pull/31558#issuecomment-1875684498 to fix https://github.com/Expensify/App/pull/31558#issuecomment-1875684498

Reply:
Should be the newest version with a patch now. It would be nice to check if the patch covers all cases 🚀

Reply:
Is there any reason for applying patch instead of using current expo-image version?
We can bump version anytime when fixed in upstream.

Reply:
There is already an upstream PR: https://github.com/expo/expo/pull/26251. Also, there might be other things fixed in the newest version so sticking to the newest one seems benefitial.